_dashboards/management/accelerate-external-data.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Optimize query performance of external data sources
+parent: Data sources
+nav_order: 30
+has_children: false
+---
+
+# Optimize query performance of external data sources
+Introduced 2.11
+{: .label .label-purple }
+
+
+
```

Comment:
SME: What limitations do we currently have?

Reply:
Limitations are mentioned here: https://github.com/opensearch-project/opensearch-spark/blob/main/docs/index.md#limitations